controller/university.go

```diff
@@ -0,0 +1,66 @@
+package controller
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/rotsg/academy-go-q32021/model"
+
+	"github.com/gorilla/mux"
+)
+
+type Controller struct {
```

Comment:
comments, also use other name as Controller is generic and when adding new controllers, this would be to much ambiguity